src/BlazorUI/Bit.BlazorUI/Components/Layouts/Footer/BitFooter.scss

```
@@ -1,31 +1,256 @@
     width: 100%;
     display: flex;
     position: relative;
-    align-items: center;
-    margin: 0.5rem 1.5rem;
+    box-sizing: border-box;
+    padding: var(--bit-ftr-padding);
+    gap: var(--bit-ftr-gap, 0);
+    flex-wrap: var(--bit-ftr-wrap, nowrap);
+    align-items: var(--bit-ftr-align, center);
+    justify-content: var(--bit-ftr-justify, normal);
+    // align-content only does anything once the content wraps, where it packs the lines the same way
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)